docs/contributor/benchmarks/otlp-logs-validation.md

```diff
@@ -0,0 +1,407 @@
+- **Generator:** 20 replicas (new set-up)
+- **Results:**
+  - Agent RECEIVED/EXPORTED: 15.3K
+  - Gateway RECEIVED/EXPORTED: 15.3K
+
+#### ⏳⭐️ 16 Jan 2025, ~13:17 (20 min)
+- Gateways on separate nodes
+- No Istio
+- **Generator:** 10 replicas
+- **Results:**
+  - Agent RECEIVED/EXPORTED: 18.8K
+  - Gateway RECEIVED/EXPORTED: 18.8K
+  - Agent Memory: 76/73
+  - Agent CPU: 0.8/0.9
+  - Gateway Memory: 69/27
+  - Gateway CPU: 0.6/0.6
+  - Gateway QUEUE: 1/0
+
+#### ⏳⭐️ 16 Jan 2025, ~13:56 (20 min)
+- No gateway involved, agent sending directly to mock backend
+- With Istio
+- **Generator:** 10 replicas
+- **Results:**
+  - Agent RECEIVED/EXPORTED: 19K
+  - Agent Memory: 82/74
+  - Agent CPU: 1.3/0.8
+
+#### 🪲 17 Jan 2025, ~10:36
+- 1 node
+- No gateway involved, agent sending directly to mock backend
+- With Istio
+- Agent has everything removed (no processors)
+- **Generator:** 5 replicas
+- **Results (without batching):**
+  - Agent RECEIVED/EXPORTED: 11.8K / instance
+- **Results (with batching):**
+  - Agent RECEIVED/EXPORTED: 14K / instance
+
+#### 🪲 17 Jan 2025, ~11:48
+- 1 node
+- No gateway involved, agent sending directly to mock backend
+- With Istio
+- Agent has everything removed (no processors), then we incrementally add them
+- **Generator:** 30 replicas (10m CPU limit)
+- 📥 Debug Exporter:
+  - **Results (without batching):**
+    - Agent RECEIVED/EXPORTED: 16K / instance
+  - **Results (with batching):**
+    - Agent RECEIVED/EXPORTED: 22.4K / instance
+  - **Results (batching + filestorage):**
+    - Agent RECEIVED/EXPORTED: 20K / instance
+- 📥 OTEL Exporter:
+  - **Results (batching + filestorage):**
+    - Agent RECEIVED/EXPORTED: 15K / instance
+  - **Results (batching + filestorage + sending queue):**
+    - Agent RECEIVED/EXPORTED: 15K / instance
+
+#### 🪲 17 Jan 2025, ~13:16
+- No gateway involved, agent sending directly to mock backend
+- With Istio
+- **2 nodes:**
+  - **Generator:** 60 replicas (10m CPU limit)
+  - Agent RECEIVED/EXPORTED: 28.6K
+  - Agent Memory: 78/71
+  - Agent CPU: 1.3/1.3
+- **3 nodes:**
+  - **Generator:** 90 replicas (10m CPU limit)
+  - Agent RECEIVED/EXPORTED: 44.6K
+  - Agent Memory: ~76-90
+  - Agent CPU: ~1.3
+
+
+## Comparison with FluentBit Setup
+In the FluentBit setup, for the very same (initial) scenario (i.e. 10 generator replicas [old set-up] / 2 agents), the [load test](https://github.com/kyma-project/telemetry-manager/actions/runs/12691802471) outputs the following values for the agent:
+- Exported Log Records/second: 27.8K
+
+
+## Conclusions
+- Before 15 Jan. (first session):
+  - Compared to the FluentBit counterpart setup, a lower performance can be expected.
+  - Backpressure is currently not backpropagated from the gateway to the agent, resulting in logs being queued/lost on the gateway end. That's because the agent has no way of knowing when to stop, thus exports data continuously (this is a known issue, which is expected be solved by the OTel community in the next half year).
```

Comment:
```suggestion
  - Backpressure is currently not backpropagated from the gateway to the agent, resulting in logs being queued or lost on the gateway end. That's because the agent has no way of knowing when to stop, thus exports data continuously (this is a known issue, which is expected be solved by the OTel community in the next half year).
```